src/main/kotlin/racingcar/controller/RacingGameApplication.kt

```diff
@@ -0,0 +1,18 @@
+package racingcar.controller
+
+import racingcar.domain.Car
+import racingcar.domain.DefaultCarMovementDecider
+import racingcar.domain.RacingGame
+import racingcar.view.InputView.inputCarNames
+import racingcar.view.InputView.inputMoveTryCount
+import racingcar.view.ResultView.printRacingResult
+import racingcar.view.ResultView.printWinners
+
+fun main() {
+    val cars = inputCarNames().map { Car(name = it) }
```

Comment:
입력 값을 `List<Car>`로 변환해주는 로직은 메서드로 뽑으면 조금 더 깔끔할 수 있겠네요. 

```suggestion
    val cars = generateCars()
```

Reply:
확실히 깔끔하네요! 반영하겠습니다🙂